reference_data/management/commands/utils/update_utils.py

```diff
@@ -90,7 +90,8 @@ def update_records(reference_data_handler, file_path=None):
 
         for line in tqdm(f, unit=" records"):
             record = dict(zip(header_fields, line.rstrip('\r\n').split('\t')))
-            for record in reference_data_handler.parse_record(record):
+            records = reference_data_handler.parse_record(record)
+            for record in records:
```

Comment:
Why change this? It makes no difference to the code execution, it doesn't make it any easier to read...

Reply:
It is weird that it doesn't work without change it with mocking tqdm. I will change it back after removing tqdm mocking.

Reply:
Changed back after removing mocked tqdm.